messages/deploy.md

```diff
@@ -18,12 +18,20 @@ Use this command for quick and simple deploys. For more complicated deployments,
 
 - Deploy a project and use stored values from a previous command run:
 
-     <%= config.bin %> <%= command.id %>
+  <%= config.bin %> <%= command.id %>
 
 - Reprompt for all deployment inputs:
 
-     <%= config.bin %> <%= command.id %> --interactive
+  <%= config.bin %> <%= command.id %> --interactive
 
 # flags.interactive.summary
 
 Force the CLI to prompt for all deployment inputs.
+
+# error.initialization
+
+One or more initialization steps failed.
+
+# error.initialization.title
+
+Initialization Failures
```

Comment:
I feel like `initialization` implies that the thing I was attempting to deploy was never able to start - but could these errors happen during the deploy? Or even after the deploy?

Reply:
From what I see in the code is the failures only being filled in during run hook, so I feel that initialization is appropriate here.

Reply:
The `Error: One or more...` message is referring to the errors listed in the table, right?  If so, I think it would be clearer to put that message first, before the table.  Using your original example, something like this:

```
Analyzing project
=== Initialization Failures

Error: One or more initialization steps failed. The following table describes each failure. 

 Plugin Name Error Name Error Message      
 ─────────── ────────── ────────────────── 
 foo         fooerror   bad stuff happened 
```

Reply:
@jshackell-sfdc The error that is displayed is the result of an exception being thrown. I need to be able to terminate the command and throwing the error is the most efficient way to do this.

Reply:
@jshackell-sfdc How about this
``` bash
Analyzing project
=== Initialization Step Failures - The following table describes each failure

 Error Name Error Message      
 ────────── ────────────────── 
 fooerror   bad stuff happened 
Error: One or more initialization steps failed.

(InitializationError)
```